arraysForDeletion.js

```diff
@@ -0,0 +1,246 @@
+function arraysForDeletion(){
+    let emptyTab = []
+    let emptyTabCopy = []
+    for (let i = 0;i<board1.size;i++){
+        let emptyEl = document.getElementById(board1.tableId[i])
+        if (emptyEl.innerHTML === ' '){                         // making an array with empty fields
+            emptyTab.push(board1.tableId[i])
+        }
+    }
+    emptyTabCopy = emptyTab                               // converting previous array to int
+    console.log(emptyTabCopy)
+    for (let i =0;i<emptyTabCopy.length;i++){
+        let strChange = emptyTabCopy[i]
+        strChange = strChange.replace("td","")
+        emptyTabCopy[i] = strChange
+        emptyTabCopy[i] = Number(emptyTabCopy[i])
+    }
+    console.log(emptyTabCopy)
+    let array1 = []
+    let arrayLeft = []
+    let arrayRight = []
+    let tempArrayForSeparation = []
+    let tempArrayLeftSeparate = []
+    let tempArrayRightSeparate = []
+    let tempArray = []
+    let tabsIntWithoutEdges = []
+    let leftArraysDone = []
+    let rightArraysDone = []
+    let leftEdgesInt = []
+    let rightEdgesInt = []
+    function getLeftAndRightEdges() {
+        let allBoard = []
+        for (let i = 0;i<board1.size;i++){
+            allBoard.push(i)
+        }
+        let addNumToArray = 0
+        leftEdgesInt.push(addNumToArray) //left edge of the board always starts with 0
```

Comment:
use airbnb linter for code rules and formatting https://www.npmjs.com/package/eslint-config-airbnb